Vagrantfile

```diff
@@ -1,7 +1,6 @@
 # -*- mode: ruby -*-
 # vi: set ft=ruby :
 load "settings.sh"
-INTERFACE = (%x(  ip route | grep -m 1 default | cut -d' ' -f5 )).strip
```

Comment:
why?

Reply:
settings.sh doesn't have INTERFACE defined.

Reply:
However, Vagrant will throw error as `INTERFACE` is not initialised. @d78ui98 Can you check.

Reply:
@akash0x53 First the user has to run `test.sh start`. It will create a virtual interface and add its name to `Vagrantfile`. 
once this done then vagrant will now throw the error.

Reply:
What Akash means, is that you should make sure that INTERFACE is always defined on `settings.sh` (not only when you run a test). Right now it is not.

Reply:
oh. Okay. Yes I can do that.